conference/react-js/src/components/ForgotPassword/forgot-password-message.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const ForgotPasswordMessage = () => (
```

Comment:
if this kind of popup message is repeatable across the application you can create generalized component which takes title, description and path(for Close button) as props

Reply:
Maybe, this will be done when we see all the components

Reply:
deal